packages/labs/react/src/create-component.ts

```diff
@@ -4,73 +4,87 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+import type React from 'react';
+
 const NODE_MODE = false;
+const DEV_MODE = true;
 
-import type React from 'react';
+/**
+ * Creates a type to be used for the props of a web component used directly in
```

Comment:
does `directly` mean "unwrapped" here?

Reply:
Correct. No wrapper and just the custom element tag in JSX.